build/tests/03-http2-admin-api.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ -n "${VERBOSE:-}" ]; then
+    set -x
+fi
+
+source .requirements
+source build/tests/util.sh
+
+kong_ready
+
+msg_test "Check if cURL supports HTTP/2"
+if ! curl --version | grep -i "http2" > /dev/null; then
+  msg_yellow "local cURL does not support HTTP/2, bypass HTTP/2 tests"
+  exit 0
```

Comment:
I feel this should be a `exit 1` instead -- any condition that leads tests to not run at all must be reported as a failure.

Reply:
ideally, yes. but i don't want this to be a blocker for this urgent release; if we find any image that we built (especially rpm ones as it's not used by most of developers, so may slip over) is failing because we can't test http2 on it.
right now let's make sure alpine is verified and consider improve this in the future.
the test stack should be rewritten with a framework that is not dependent on the artifact that we are releasing (i.e. testing tools running outside of docker), and more developer friendly error reporting

Reply:
I changed this from bypass to fail - since we're running the smoke test on Ubuntu 22.04 and the curl on it supports HTTP/2 by default.

A scenario that does not support HTTP/2 fails on this script also makes sense.

Reply:
@windmgc i saw you made this change already, if so, please make sure all at least the images in our smoke-tests suites does have http2 support in curl.